citus.spec

```diff
@@ -1,17 +1,17 @@
-%global pgmajorversion 10
-%global pgpackageversion 10
+%global pgmajorversion 11
```

Comment:
Do we still need pgmajorversion check at the line 66 ?

Reply:
I guess we don't. I'll remove that unnecessary check before merging.